README.md

```diff
@@ -53,6 +53,25 @@ storiesOf('Button')
 
 > Have a look at [this example](example/story.js) stories to learn more about the `addWithInfo` API.
 
+## Use with Docgen
+We may add the description to prop. For example:
```

Comment:
> We may add the description to prop.

I'm not really sure what this means. Can you clarify and change the wording?

(No worries if you're not a native English speaker, I can help out!)